pkg/workload/workload_test.go

```diff
@@ -110,6 +142,46 @@ func TestNewInfo(t *testing.T) {
 							corev1.ResourceMemory: 3 * 1024 * 1024,
 							"ex.com/gpu":          3,
 						},
+						Count: 3,
+					},
+				},
+			},
+		},
+		"admitted with reclaim": {
+			workload: *utiltesting.MakeWorkload("", "").
+				PodSets(
+					*utiltesting.MakePodSet("main", 5).
+						Request(corev1.ResourceCPU, "10m").
+						Request(corev1.ResourceMemory, "10Ki").
+						Obj(),
+				).
+				Admit(
+					utiltesting.MakeAdmission("").
+						Assignment(corev1.ResourceCPU, "f1", "40m").
+						Assignment(corev1.ResourceMemory, "f1", "40Ki").
+						AssignmentPodCount(4).
```

Comment:
In the real world, `AddignmentPodCount` is `3`, right?

Reply:
done